frontend/src/constants/theme.ts

```diff
@@ -37,7 +37,12 @@ const themeColors = {
 	matterhornGrey: '#555555',
 	whiteCream: '#ffffffd5',
 	black: '#000000',
+	gainsboro: '#DBDBDB',
+	navyBlue: '#1668DC',
 	lightgrey: '#ddd',
+	lightSkyBlue: '#8DCFF8',
```

Comment:
do check we already have blue in antd colors

Reply:
Yes i checked already @palashgdev , But this shade of color as per figma is not present in antd color.